.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Kiali-UI CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12' ]
+    name: Node ${{ matrix.node }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - run: yarn install
```

Comment:
I don't know if there is a way to personalize which version of yarn to use.
If the default one is good enough, it's ok, but I remember that this created problems in the past and travis installs a specific version.

Reply:
Note, this command is not installing yarn, I guess yarn comes from the node image

Reply:
Node 14 image uses:
```
Run yarn install
yarn install v1.22.5
```
And travis installs

```
--version 1.22.4
```

So, perhaps it's ok.

Reply:
Evidently, there is a yarn version implicit to the image and that seems to work. I started out using a yarn action because I thought that I had to and that caused various issues in running the tests (related to ES2019 compatibility). I spent over a day trying to figure that one out.